static/app/views/insights/pages/frontend/frontendOverviewPage.tsx

```diff
@@ -47,14 +49,23 @@ import {
   ProjectPerformanceType,
 } from 'sentry/views/performance/utils';
 
+const DURATION_TOOLTIP = tct(
+  'A heuristic measuring when a pageload or navigation completes. Based on whether the initial load of the webpage has become idle [link:Learn more.]',
+  {
+    link: (
+      <ExternalLink href="https://docs.sentry.io/platforms/javascript/tracing/instrumentation/automatic-instrumentation/#idletimeout" />
+    ),
+  }
+);
+
 export const FRONTEND_COLUMN_TITLES = [
   'transaction',
   'operation',
   'project',
   'tpm',
-  'p50()',
-  'p75()',
-  'p95()',
+  {title: 'p50()', tooltip: DURATION_TOOLTIP},
+  {title: 'p75()', tooltip: DURATION_TOOLTIP},
+  {title: 'p95()', tooltip: DURATION_TOOLTIP},
```

Comment:
Personally, i'm not a fan of the `React.ReactNode | ColumnTitle` stuff.
It causes you to have to do some funky stuff like calling the `isColumnTitle` function. Instead I think column titles should always be `ColumnTitle[]`.

But I get why you might do this, so much to refactor if you enforced that. So I think it would be best to leave a TODO comment to update this in the future.